projects/core/directives/hint/hint-manual.directive.ts

```diff
@@ -12,19 +13,22 @@ import {TuiHintHover} from './hint-hover.directive';
 })
 export class TuiHintManual extends TuiDriver implements OnChanges {
     private readonly hover = inject(TuiHintHover);
-    private readonly stream$ = new BehaviorSubject(false);
+    private readonly stream$ = new BehaviorSubject<boolean | null>(false);
```

Comment:
You shouldn't change the type, just use `!!` to cast `null` to `false` when you call `next`.